Add Contact Form Section

luisrpavanelli opened the pull request:
In this PR, we worked add contact form section, in html and css files, with collaborating @smunoz1988 and @Mylo16.

We modified the html and css files, and also implemented formspree.

Index.html and syle.css:
- create a new brand with my collaborators
- create a contact form code in html and css according to figma
- correction of linters


formspree:
- create a formspree account
- create a code
- create and modify index.hmtl and style.css to place the formspree

Qoosim reviewed (commented):
#  *Status: Invalid Request* :red_circle: 

Hi @team, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that 

- [ ] reflects commit from only one partner in your pair-programming project. Every members of the team must at least have one commit in a group project. Kindly ensure every member contributes to the project with a commit message.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

luisrpavanelli commented:
In this PR, we worked add contact form section, in html and css files, with collaborating @smunoz1988 and @Mylo16.

shella12 commented on style.css:
```diff
@@ -227,3 +227,28 @@ hr.roundeddivider {
   width: 50px;
   margin: 24px auto;
 }
+
+form {
+  margin: 10px auto;
+  width: 463px;
+  height: 256px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 10px;
+  line-height: 20px;
+
+  /* or 173% */
+
+  display: flex;
+  align-items: center;
+
+  /* Neutrals/N800 */
+
+  color: #172b4d;
+
+  /* Inside auto layout */
+
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
```
- [ ] Please make sure that your `contact us` section looks like the provided [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468&t=OrdN01rgF7wtPtkc-0) as it is the [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md#project-requirements) of the project

> You don't need to implement any functionality that requires JavaScript.
You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).
Implement the button interactions (enable, hover, etc.).

_**Figma design**_ 

<img width="221" alt="Screenshot 2023-02-22 at 3 26 58 AM" src="https://user-images.githubusercontent.com/44798044/220472900-d45c6d8c-286b-4141-bf1b-6e16dff024bf.png">

_**Your design**_

<img width="363" alt="Screenshot 2023-02-22 at 3 18 16 AM" src="https://user-images.githubusercontent.com/44798044/220472936-eaf83e43-84e1-410f-a66c-dfc3f7a971de.png">

shella12 commented on index.html:
```diff
@@ -233,5 +233,14 @@ <h3>Skills</h3>
+    <section>
+        <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+        <form action="">
+            <input type="text" name="user_name" placeholder="Full name" id="name" required>
```
- [ ] `Name` field input should not accept more than 30 characters as it is mentioned in  [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md#project-requirements) 

> Name field accepts up to 30 chars.

Kindly add  `maxlenght=30` attribute in the name text input field

shella12 commented on index.html:
```diff
@@ -233,5 +233,14 @@ <h3>Skills</h3>
+        <form action="">
+            <input type="text" name="user_name" placeholder="Full name" id="name" required>
+            <input type="email" name="mail" placeholder="Email address" id="email" required>
+            <textarea name="message" id="msg" cols="30" rows="10" required placeholder="Enter text here"></textarea>
```
- [ ] Please make sure all input fields including the message field have `required` attribute set as it is the [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md#project-requirements) of the project

> A user cannot submit the form if one or more of the fields is empty.

- [ ] Kindly set `maxlength = 500` attribute for `textarea` message field as mentioned in the project [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md#project-requirements) of the project

> Message field accepts up to 500 chars.

shella12 commented on index.html:
```diff
@@ -233,5 +233,14 @@ <h3>Skills</h3>
     </section>
+    <section>
+        <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+        <form action="">
```
- [ ] As stated in the [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md#project-requirements). you should use  `formspree` to collect data from your `contact` form 

> Collect data by using the [Formspree service](https://formspree.io/html).

Please make sure `Form` uses the action and method attribute to integrate with [formspree service](https://formspree.io/html).

- `action` value should be similar to `https://formspree.io/f/{form_id}`

- `method` value equals `POST`

shella12 reviewed (changes requested):
Hi @luisrpavanelli and @smunoz1988 , 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!

<img src="https://media1.giphy.com/media/Jq824R93JsLwZCaiSL/giphy.gif"/>

## Highlights
- All linters passed ✅ 
- All inputs are inclosed in `form` ✅ 

## Required Changes ♻️

- [ ] _Check the comments under the review._


## Optional suggestions

- [ ] [OPTIONAL] Please add all contributors in to the author section of your README to give credit where credit is due


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

luisrpavanelli commented:
In this PR, we worked add contact form section, in html and css files, with collaborating @smunoz1988 and @Mylo16.

We modified the html and css files, and also implemented formspree.

Index.html and syle.css:
- create a new brand with my collaborators
- create a contact form code in html and css according to figma
- correction of linters


formspree:
- create a formspree account
- create a code
- create and modify index.hmtl and style.css to place the formspree

souad988 commented on index.html:
```diff
@@ -233,5 +233,61 @@ <h3>Skills</h3>
+        <div class="contactformcontainer">
+            <div class="contactcard">
+                <div class="text">
+                    <input type="text" name="user_name" placeholder="Full name" id="name" required>
```
[ optional ] kindly add max-length attribute with a value 30 as requested in this project requirements

souad988 commented on index.html:
```diff
@@ -233,5 +233,61 @@ <h3>Skills</h3>
+                </div>
+                
+                <div class="message"></div>        
+                    <textarea name="message" id="msg" cols="30" rows="10" required placeholder="Enter text here"></textarea>
```
[ optional ] same here please don't forget to specify a text length limit as requested in this activity.

souad988 commented on style.css:
```diff
@@ -227,3 +227,83 @@ hr.roundeddivider {
+  margin-top: 24px;
+}
+
+.contactformcontainer:hover {
+  box-shadow: 0 5px 10px rgba(54, 179, 127, 0.8);
+}
+
+.contactformcontainer:active {
+  background-color: #008552;
+  box-shadow: none;
+}
```
I think you forgot to specify here that this properties should apply to the button not the container ,kindly check again and fix it also make sure to use the same shadow as in figma template  "0px 8px 16px rgba(54, 179, 127, 0.24)"

![student shadow](https://user-images.githubusercontent.com/59707859/220706218-4a4661d3-f098-4111-9b4a-5d5f08874b3c.PNG)

souad988 reviewed (changes requested):
Hi @luisrpavanelli , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

![212385281-4797c58a-d358-423a-aa7c-832c22203865](https://user-images.githubusercontent.com/59707859/220706466-94c92476-1263-4fa0-99b8-c0b75ef8344d.gif)

## Required Changes ♻️
_Check the comments under the review._
_ Also please don't forget to fix styling of your inputs as requested by the previous reviewer 

**your inputs:**
![temp2 student contact form](https://user-images.githubusercontent.com/59707859/220706797-c125e987-ba5c-4fab-b455-7c376424972f.PNG)

**figma design**
![figma temp2](https://user-images.githubusercontent.com/59707859/220706862-b987fe99-71f6-4a50-bd84-f137006b928e.PNG)


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

luisrpavanelli commented:
Hello, I worked to make the modifies as you tell me to do, thank you so much!

GedeonTS reviewed (approved):
# STATUS: APPROVED :tada: :clap: :champagne: 
Hi @luisrpavanelli ,

Your are amazing :smiley: :clap: !  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<img src="https://media.giphy.com/media/BPJmthQ3YRwD6QqcVD/giphy.gif"> 

## Highlights :clap: :tophat: 
- No linter errors :+1: 
- Good use of GitHub flow :+1: 
- The form is working fine :+1: 
## Optional suggestions :small_airplane: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
